daemon/monitor.go

```diff
@@ -179,9 +179,6 @@ func (m *containerMonitor) Start() error {
 			}
 			continue
 		}
-		if exitStatus.OOMKilled {
-			m.container.logEvent("oom")
```

Comment:
there's another logEvent("oom") at https://github.com/docker/docker/pull/16235/files#diff-c43715aa07d90f284fe1a3749983808bR166
should you remove this also? are tests green with this still left?